ci_framework/roles/artifacts/tasks/edpm.yml

```diff
@@ -4,20 +4,21 @@
     - not ansible_facts.services is defined
   ansible.builtin.service_facts:
 
-- name: Check for virtualized compute
+- name: Check for virtualized compute in Baremetal jobs
   when:
     - "'libvirtd' in ansible_facts.services"
   block:
     - name: List all of the existing virtual machines
       register: vms_list
+      become: true
```

Comment:
as discussed, `become: true` isn't needed since we pass the `uri: qemu:///system`.
The actual issue is the condition on L9, it should check for `libvirtd.service` instead. We may want to also ensure it's running, but we may hit some issues with the modular libvirt sooner or later - especially since the ci-framework has full support for that. Up to you - but that's the actual issue.

Reply:
Done.

Reply:
please remove that  `become: true`